toxcore/tox.h

```diff
@@ -564,9 +564,6 @@ struct Tox_Options {
      *
      * This member is ignored (it can be NULL) if proxy_type is
      * TOX_PROXY_TYPE_NONE.
-     *
-     * The data pointed at by this member is owned by the user, so must
-     * outlive the options object.
```

Comment:
Since ownership now depends on an experimental option, simply removing this wont do.

Reply:
True, I've put it back.